lnrpc/rpc.proto

```diff
@@ -1283,6 +1283,29 @@ message HTLC {
     uint32 expiration_height = 4;
 }
 
+enum CommitmentType {
+    /**
+    A channel using the legacy commitment format having tweaked to_remote
+    keys.
+    */
+    LEGACY = 0;
+
+    /**
+    A channel that uses the modern commitment format where the key in the
+    output of the remote party does not change each state. This makes back
```

Comment:
nit: maybe (but i'm also a bit unsure) "does not change upon state changes"

Reply:
Sticking with this, as that was the original doc on `static_remote_key`